### DISCUSSION TITLE
Can I synchronise a local calendar in Thunderbird to a Google calendar?

### DISCUSSION THREAD

**Discussion by David-A-James** [kewisch/gdata-provider#841] [Support] [answered]
I hoped this addon would allow synchroisation from a local calendar in Thunderbird to a Google calendar.

I have added some Google calendars to Thunderbird and they have synchronised so that I can see the events in the Google calendars in Thunderbird.

But what I want to do is synchronise the local calendar "Home" (which contains events from July 2011 onwards) to a Google calendar so that inthe future I can still use the Thunderbird local calendar as my primary calendar, but can also see the events on other devices via a Google calendar.

But the drop-down menu on Home in the Thumderbird calendar window doesn't contain a "Synchronise" item (which it does for the Google calendars).

![image](https://github.com/user-attachments/assets/5beb8dcb-03af-4871-be95-ed77013f4f5c)

This is with Thunderbird "128.5.2esr (64-bit)" on Windows 10, version "128.0.0" of "Provider for Google Calendar".

I'm probably not seeing something obvious, but ... is it possible to synchronise from a local calendar in Thunderbird to a Google calendar (and if so, how)?

**Comment by kewisch** [comment, marked as answer]
I think your best choice here would be to export the Home calendar to the ICS format and them import that into Google Calendar. I'd recommend to use the Google Calendar web interface for the import, to avoid multiple API calls. Once you've done that and synchronize in the Provider for Google Calendar, verify all your events are there and you can delete your home calendar.

**Reply by esAlphaBetaOmega** [reply]
But this is point in time / the ics exports only what was there up to export / import into google calendar.. if I add a new event in Thunderbird in the email client and calendar lightning interface, it doesn't sync to google.
Is there no dynamic sync solution? The Google calendar Provider extension is said to sync in both directions but only syncs google > thunderbird.. I see many others have this issue in various threads and fora, surely getting tbird fully mobile.. to fly!, is worth a roadmap inclusion!
